ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -780,7 +784,21 @@ def array_indices_axis(
     indices = indices[0]
     if disable_random_axis:
         return [x_dtype, indices_dtype], x, indices
-    return [x_dtype, indices_dtype], x, indices, axis, batch_dims
+
+    if values is None:
+        return [x_dtype, indices_dtype], x, indices, axis, batch_dims
+    else:
```

Comment:
Hello @PM-23, I'd suggest moving this logic to a composite strategy in the test file, instead of having to change the strategy itself, we shouldn't be changing this one here for a single test.

Reply:
Hi @CatB1t so could you please let me know I'm confused so to make it clear I need to update the code in only test file and not in helper file? But as it needs to have **values** parameter that needs to be returned when put_along_axis function is used. So can you provide me some more information for your suggestion. Thanks!

Reply:
Hi @CatB1t could you please have a look again?

Reply:
@PM-23, in that case we should try to define our composite strategy with a different behavior in the test file of the function using the `@st.composite` decorator, You can use the strategies defined here: https://unify.ai/docs/ivy/docs/helpers/ivy_tests.test_ivy.helpers.hypothesis_helpers.html to help you build your custom strategy.